modules/serverless-install-kafka-odc.adoc

```diff
@@ -0,0 +1,34 @@
+// Module is included in the following assemblies:
+//
+// serverless/serverless-kafka.adoc
+
+[id="serverless-install-kafka-odc_{context}"]
+= Installing Apache Kafka components using the web console
+
+Cluster administrators can enable the use of Apache Kafka functionality in a {ServerlessProductName} deployment by instantiating the KnativeKafka custom resource definition provided by the *Knative Kafka* {ServerlessOperatorName} API.
+
+.Prerequisites
+* You must have access to an AMQ Streams cluster.
+* You must have installed the {ServerlessOperatorName} and Knative Eventing.
+* You must have cluster administrator permissions on the {product-title} cluster.
+* You must have logged in to the web console.
+
+.Procedure
+
+. In the *Administrator* perspective, navigate to *Operators* → *Installed Operators*.
+. Check that the *Project* dropdown at the top of the page is set to *Project: knative-eventing*.
+. Click *Knative Kafka* in the list of *Provided APIs* for the {ServerlessOperatorName} to go to the *Knative Kafka* tab.
++
+image::knative-kafka-api.png[OpenShift Serverless Operator provided APIs]
+. Click *Create Knative Kafka*.
+. In the *Create Knative Kafka* page, you can configure the *KnativeKafka* object, by using either the default form provided, or by editing the YAML.
+.. Using the form is recommended for simpler configurations that do not require full control of `KnativeKafka` object creation.
+.. Editing the YAML is recommended for more complex configurations that require full control of `KnativeKafka` object creation. You can access the YAML by clicking the *Edit YAML* link in the top right of the *Create Knative Kafka* page.
+. After you have completed any optional configurations for Kafka, click *Create*.
+. You will be automatically directed to the *Knative Kafka* tab, where you will see `knative-kafka` in the list of resources.
```

Comment:
Suggest: `knative-kafka`/*knative-kafka* since it is something a user sees on the UI. Ditto, for L 31